geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/AlterQueryServiceCommand.java

```diff
@@ -0,0 +1,209 @@
+import org.apache.geode.internal.cache.InternalCache;
+import org.apache.geode.management.cli.CliMetaData;
+import org.apache.geode.management.cli.SingleGfshCommand;
+import org.apache.geode.management.internal.cli.AbstractCliAroundInterceptor;
+import org.apache.geode.management.internal.cli.GfshParseResult;
+import org.apache.geode.management.internal.cli.functions.AlterQueryServiceFunction;
+import org.apache.geode.management.internal.cli.functions.CliFunctionResult;
+import org.apache.geode.management.internal.cli.result.model.ResultModel;
+import org.apache.geode.management.internal.security.ResourceOperation;
+import org.apache.geode.security.ResourcePermission;
+
+public class AlterQueryServiceCommand extends SingleGfshCommand {
+
+  static final String COMMAND_NAME = "alter query-service";
+  static final String COMMAND_HELP =
+      "Alter configuration parameters for the query service";
+  static final String METHOD_AUTHORIZER_NAME = "method-authorizer";
+  static final String METHOD_AUTHORIZER_NAME_HELP =
+      "The name of the class to be used for OQL method authorization";
+  static final String AUTHORIZER_PARAMETERS = "authorizer-parameters";
+  static final String AUTHORIZER_PARAMETERS_HELP =
+      "A comma separated list of all parameter values for the specified method authorizer. Requires '--"
+          + METHOD_AUTHORIZER_NAME + "' option to be used";
+  static final String SINGLE_AUTHORIZER_PARAMETER = "authorizer-parameter";
+  static final String SINGLE_AUTHORIZER_PARAMETER_HELP =
+      "A single parameter to be used by the specified method authorizer. Requires '--"
+          + METHOD_AUTHORIZER_NAME + "' option to be used";
+  static final String DEPRECATED_PROPERTY_ERROR = "Deprecated System Property: \"" + GEMFIRE_PREFIX
+      + "QueryService.allowUntrustedMethodInvocation\" is set to TRUE. In order to use a MethodInvocationAuthorizer, this property must be FALSE or undefined.";
+  static final String PARAMETERS_WITHOUT_AUTHORIZER_MESSAGE =
+      "The '--" + AUTHORIZER_PARAMETERS + "' and '--" + SINGLE_AUTHORIZER_PARAMETER
+          + "' options require '--"
+          + METHOD_AUTHORIZER_NAME + "' to be specified";
+  static final String SINGLE_PARAM_AND_PARAMETERS_SPECIFIED_MESSAGE =
+      "Only one of '--" + AUTHORIZER_PARAMETERS + "' and '--" + SINGLE_AUTHORIZER_PARAMETER
+          + "' options can be specified.";
+  static final String NO_ARGUMENTS_MESSAGE =
+      "No arguments were provided. No changes have been applied.";
+  static final String NO_MEMBERS_FOUND_MESSAGE = "No members found.";
+  static final String SECURITY_NOT_ENABLED_MESSAGE =
+      "Integrated security is not enabled for this distributed system. Updating the method authorizer requires integrated security to be enabled.";
+  public static final String PARTIAL_FAILURE_MESSAGE =
+      "In the event of a partial failure of this command, re-running the command or restarting failing members should restore consistency.";
```

Comment:
Some constants are not used outside of the package, can you make them `private`?.

Reply:
Done!